opensearch-operator/controllers/suite_test.go

```diff
@@ -85,13 +89,17 @@ var _ = BeforeSuite(func() {
 	Expect(err).NotTo(HaveOccurred())
 	err = opsterv1.AddToScheme(scheme.Scheme)
 	Expect(err).NotTo(HaveOccurred())
+	err = prometheus.AddToScheme(scheme.Scheme)
+	Expect(err).NotTo(HaveOccurred())
 
 	k8sClient, err = client.New(cfg, client.Options{Scheme: scheme.Scheme})
 	Expect(err).NotTo(HaveOccurred())
 	Expect(k8sClient).NotTo(BeNil())
 
 	k8sManager, err := ctrl.NewManager(cfg, ctrl.Options{
-		Scheme: scheme.Scheme,
+		Scheme:                 scheme.Scheme,
```

Comment:
Hey @swoehrl-mw
can you please explain why you add the `MetricsBindAddress` and `HealthProbeBindAddress` ?

Reply:
By adding both I can configure them with random ports so there are no collisions. I just copied that part from the other package so they behave the same and because I ran into a collision while testing locally.